post-deployment/openstack/logging.yml

```diff
@@ -50,7 +35,7 @@
          name: "elasticsearch-operator"
          namespace: "openshift-operators-redhat"
        spec:
-         channel: "{{ openshiftVersion }}"
+         channel: "stable"
```

Comment:
I think we could do something like this `"{{ "stable" if openshiftVersion >= 4.7 else openshiftVersion }}"` which should work with 4.6 as well as 4.7+
Needs to be tested though...

Reply:
Done. And tested in v4.7.7 and 4.6.9